emscripten.py

```diff
@@ -648,14 +648,17 @@ def include_asm_consts(pre, forwarded_json, metadata, settings):
 
   asm_consts, all_sigs = all_asm_consts(metadata)
   asm_const_funcs = []
-  for sig in set(all_sigs):
-    forwarded_json['Functions']['libraryFunctions']['_emscripten_asm_const_' + sig] = 1
+  for call_type, sig in set(all_sigs):
```

Comment:
This will all need to be repeated in `create_asm_consts_wasm`. Good opportunity to extract more common parts?

Reply:
Not sure what to extract here. Refactoring the code a little, as requested in your comment below as well.